docs/admin-sdk/docs/guide/2_api-reference/ui/mediaModal.md

```diff
@@ -0,0 +1,33 @@
+# Media modal
```

Comment:
Can you add a information since which Shopware version this feature is available? Otherwise it looks good 👍